projects.json

```
@@ -2036,6 +2081,66 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/DeclarativeHub/ReactiveKit.git",
+    "path": "ReactiveKit",
+    "branch": "master",
+    "compatibility": [
+      {
+        "version": "4.2",
+        "commit": "fcf4a94d8e0cdd4e312286668340609bb54ff417"
```

Update hash to newer commit.
```suggestion
        "commit": "1e79f6ca02f6243aae1e46335dfc28c0313f01a1"
```